client/state/plugins/installed/selectors-ts.ts

```diff
@@ -228,6 +228,15 @@ export const getFilteredAndSortedPlugins = createSelector(
 	}
 );
 
+export function getPluginsWithUpdates( state: AppState, siteIds: number[] ) {
+	return getFilteredAndSortedPlugins( state, siteIds, undefined )
+		.filter( _filters.updates )
+		.map( ( plugin ) => ( {
+			...plugin,
+			type: 'plugin',
```

Comment:
Just verifying that the `version` is intentionally missing?

Reply:
Yes, `version` is found on the objects in the `sites` parameter now. This is one of the things that was causing issues before. A plugin itself doesn't have a version, it is a installation of a plugin on a particular site that has a version. A plugin could have a "latest version", but the old version property wasn't that. It was just the version that the plugin was installed as on the last site in the array of sites, which created some inconsistency.